catroid/src/org/catrobat/catroid/pocketmusic/note/trackgrid/TrackToTrackGridConverter.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2016 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.pocketmusic.note.trackgrid;
+
+import android.util.SparseArray;
+
+import org.catrobat.catroid.pocketmusic.note.MusicalBeat;
+import org.catrobat.catroid.pocketmusic.note.NoteEvent;
+import org.catrobat.catroid.pocketmusic.note.NoteLength;
+import org.catrobat.catroid.pocketmusic.note.NoteName;
+import org.catrobat.catroid.pocketmusic.note.Track;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TrackToTrackGridConverter {
+
+	public TrackToTrackGridConverter() {
+	}
+
+	public TrackGrid convertTrackToTrackGrid(Track track, MusicalBeat beat, int beatsPerMinute) {
```

Comment:
The `TrackToTrackGridConverter` is a (stateless) utility class. Make the [constructor private](https://github.com/Catrobat/Catroid/blob/develop/catroid/src/org/catrobat/catroid/utils/Utils.java#L88-L91) and the method static.